### PR TITLE
Add response_time to monit plugin

### PR DESCRIPTION
**Pull request by wowi42**
### Required for all PRs:

- [X] Signed [CLA](https://influxdata.com/community/cla/).
- [X] Associated README.md updated.
- [X] Has appropriate unit tests.

### CODE REVIEW

**Review by soufiane-bouchaara — APPROVED** [review]
Looks good to me , I was waiting for this update

**Comment by sjwang90** [comment]
@SirishaGopigiri if you'd like to take a look.

**Comment by wowi42** [comment]
not sure if the failed test is linked to my update...

**Comment by srebhan** [comment]
Please rebase on master to fix the AppVeyor problem.

**Comment by wowi42** [comment]
@srebhan I think it's ok now, right?

**Comment by srebhan** [comment]
@wowi42 if it builds it's ok! ;-)

**Comment by wowi42** [comment]
@srebhan It seems so, no?

**Comment by srebhan** [comment]
Yes, it's building ok now.

**Comment by wowi42** [comment]
Any plan to merge it?

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,5 @@
-ifeq ($(OS), Windows_NT)
-	devnull := nul
-else
-	devnull := /dev/null
-endif
-
 next_version := 1.16.0
-tag := $(shell git describe --exact-match --tags 2>$(devnull))
+tag := $(shell git describe --exact-match --tags 2>git_describe_error.tmp; rm -f git_describe_error.tmp)
```
Comment (ssoroka) [inline Makefile:2]: this should not be changed in your PR.

**Review by ssoroka — CHANGES_REQUESTED** [review]
There seems to be something wrong with this PR, it has a bunch of unrelated changes.

File: Makefile (hunk 1 above)
Comment (srebhan) [reply Makefile:2]: @wowi42: Rebasing on the latest master should remove this I think.

**Comment by wowi42** [comment]
fixed @ssoroka

**Review by srebhan — APPROVED** [review]
@ssoroka seems like the unrelated changes are gone. LGTM.

**Review by ssoroka — APPROVED** [review]

**Comment by ssoroka** [comment]
Thank you! merged

**Pull request merged**